docs/WritingTests.asciidoc

```diff
@@ -191,25 +191,29 @@ clone_job.pl --from localhost 42 _GROUP=0
 === Using snapshots to speed up development of tests
 [id="snapshots"]
 
-Sometimes it's annoying to run the full installation to adjust some
-test. It would be nice to have the VM jump to a certain point. +QEMU
-backend+ provides feature that allows a job to start from a snapshot that might
-help in this situation.
+Sometimes it is annoying to run the full installation to adjust some
+test. It would be nice to have the VM jump to a certain point. The +QEMU
+backend+ provides a feature that allows a job to start from a snapshot which
+can help in this situation.
 
 Depending on the use case, there are two options to help:
 
-* create and *preserve* snapshots for *every test* module run (+MAKETESTSNAPSHOTS+)
-  - offers more flexibility as test can be resumed almost at any point, however
-    disk space requirements are high (expect more than 30GB for one job)
-  - this mode is useful for fixing non fatal issues in tests and debugging SUT
+* Create and *preserve* snapshots for *every test* module run (+MAKETESTSNAPSHOTS+)
+  - Offers more flexibility as the test can be resumed almost at any point.
+    However disk space requirements are high (expect more than 30GB for one
+    job)
+  - This mode is useful for fixing non-fatal issues in tests and debugging SUT
```

Comment:
Well, the sentence makes sense but I added one sentence for extension.

@asmorodskyi it does not state that it is not useful for fixing fatal issues but only that it is useful for fixing non-fatal. Take for example the following test plan
* installation (many modules)
* boot_to_desktop (passed)
* console1 (failed, not-important module)
* console2 (passed)

using the second mode with TESTDEBUG the snapshot from *console2* would overwrite the one from *boot_to_desktop* and therefore one can not go back to *boot_to_desktop* to debug *console1*